GitUI/CommandsDialogs/SettingsDialog/Pages/AppearanceSettingsPage.cs

```diff
@@ -93,6 +93,8 @@ protected override void SettingsToPage()
 
             chkShowRelativeDate.Checked = AppSettings.RelativeDate;
 
+            chkShowGpgInformation.Checked = AppSettings.ShowGpgInformation;
```

Comment:
I am not sure if this setting should go to this page. I would see the Appearance page as more general, applying to the whole app. I know that now there are general settings mixed with specific ones. The suggested page for this setting is `Detailed -> Browse repository window`. I am also open for other proposals.

Reply:
We probably need to review the whole menu structure. It is a separate discussion, of course.

Reply:
Yeah, there was an attempt for that https://github.com/gitextensions/gitextensions/issues/1707

Reply:
yes, thanks, I did come across it trawling the history.
i haven't had time to formally go through it. i'm just flagging it may be time to review it again.

Reply:
For now I leave it there, Visual Studio's graphical editor did not "collaborate" when I created the menu option :( 
Add also this as point on your menu restyling discussion.

Reply:
https://github.com/gpongelli/gitextensions/pull/5